### DISCUSSION TITLE
[组队]去中心化空投平台

### DISCUSSION THREAD

**Discussion by MoveCatcher** [NonceGeek/Move-Camp#171] [Amazing Things!]
💡一句话 IDEA：
一个平台, 给项目方快速空投到真实用户, 给用户做任务撸毛

我的角色：前端 + move 合约

人数需求（不包含发起者）：move 1人

我的联系方式：tg: @roger_tim

个人简介：move 开发, 前端开发

**Comment by pingp76** [comment]
tg没有搜到
